library/lib_exec_node.ks

```diff
@@ -0,0 +1,43 @@
+@lazyglobal off.
+
+function MaxShipThrust{ 
+	// this function is needed as a workaround
+	// to kOS bug regarding ship thrust.
+	local engs is list().
+	list engines in engs.
+	local mth is 0.
+	for eng in engs{
+		if eng:VISP > 0 {
+			set mth to mth + eng:MAXTHRUST.
+		}
+	}
+	return mth.
+}
+
+function exec_node{
+	parameter nd.
+	
+	local mth is MaxShipThrust().
+	lock burntime to nd:deltav:mag/(mth/mass).
+
+	lock steering to nd.
+	wait until nd:eta<burntime.
+
+	lock throttle to cos(vang(nd:deltav,ship:facing:vector)).
+	// Thanks to cosine, we burn only when we face the correct direction.
+
+	wait until burntime<5. // last 5 seconds of burn should be slower
+	local scale is nd:deltav:mag.
+
+	lock throttle to nd:deltav:mag/scale*
+		cos(vang(nd:deltav,ship:facing:vector)).
+	// We burn slower and slower.
+
+	wait until nd:deltav:mag<scale*0.01.
+	lock throttle to 0.
+
+	unlock throttle.
+	unlock steering.
+}
+
+exec_node(nextnode).
```

Comment:
Excuse me. It was too loud. But this line breaks the idea of `lib_` prefix. That's how I understand what `lib_` prefix means: "`run lib_foo.` at the start of your program to import functions from the library and then use those functions". But this library assumes that you will import it to execute the next maneuver node. It's very implicit if you ask me.